lib/importer/csv_manifest.rb

```diff
@@ -3,7 +3,7 @@ class CSVManifest
     include ActiveModel::Validations
 
     def self.controlled_vocabularies
-      @controlled_vocabs ||= Qa::Authorities::Local.names
+      ["licenses", "rights_statements", "resource_types"]
```

Comment:
Might be good to put this value in a centralized config setting, but that could be done later.